bots/elasticsearch/tasks.py

```diff
@@ -219,12 +238,12 @@ def pseudo_bisection(self, es_url, es_index, min_date, max_date, dry=False):
             return
 
         logger.debug('dry=False, reindexing missing works')
-        bot = apps.get_app_config('elasticsearch').get_bot(
-            ShareUser.objects.get(username=settings.APPLICATION_USERNAME),
-            es_filter={'date_created__range': [min_date, max_date]},
-        )
-        bot.run()
-        return
+
+        task = update_elasticsearch(filter={
+            'date_created__range': [min_date, max_date]
+        })
+
+        return logger.info('Spawned %r', task)
```

Comment:
I agree naked returns look weird, but this seems to imply something meaningful is returned. I think we should avoid it.